src/arch/riscv/pmp/pmp.h

```diff
@@ -60,14 +53,14 @@ void set_pmp_entry(unsigned int *index, unsigned int flags, uintptr_t base, size
  * @param reg The index of the PMP configuration register to write to.
  * @param value The value to write to the register.
  */
-static inline void write_pmpcfg(uint32_t reg, uint32_t value);
+void write_pmpcfg(uint32_t reg, uint32_t value);
```

Comment:
please use 'exten' for declarations

Reply:
Okay, I will use extern and commit the changes in a few minutes.